packages/wonder-blocks-modal/src/util/find-focusable-nodes.ts

```diff
@@ -2,8 +2,7 @@
  * List of elements that can be focused
  * @see https://www.w3.org/TR/html5/editing.html#can-be-focused
  */
-const FOCUSABLE_ELEMENTS =
-    'a[href], details, input, textarea, select, button:not([aria-label^="Close"])';
+const FOCUSABLE_ELEMENTS = "a[href], details, input, textarea, select, button";
```

Comment:
**note:** This is the main change in this PR. By removing that custom selector, we can include the dismiss button in the collection.